features/jest/client/pom.xml

```diff
@@ -92,6 +92,11 @@
       <version>2.14.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.github.resilience4j</groupId>
+      <artifactId>resilience4j-circuitbreaker</artifactId>
+      <version>${resilience4jVersion}</version>
```

Comment:
The used version here is very old (0.17.0). Can it be upgraded to 2.0.0?

Reply:
Looks like 2.0 works only with java 17

Reply:
1.7?
it is compatible with java v. 1.8

Reply:
Thats looks ok to me